Basics/3.cpp

```diff
@@ -0,0 +1,18 @@
+//Write a program to check whether a number is even or odd
+#include<iostream>
+using namespace std;
+int main()
+{
+    int n;
+    cout<<"enter a number";
+    cin>>n;
+    if(n%2==0)
+    {
+	cout<<n<<" is even";
+    }
+    else
+    {
+	cout<<n<<" is odd";
+    }
+    return 0;
+}
```

Comment:
All these Basics/1.cpp,Basics/2.cpp etc needs to be deleted as well before approval for merging.

Reply:
i have deleted the basics and added milestone folder under my name folder .